Source/WebKit/UIProcess/API/gtk/WebKitPrintOperation.cpp

```diff
@@ -74,6 +80,22 @@ enum {
     LAST_SIGNAL
 };
 
+class PreparePrintResponse {
```

Comment:
Maybe it woulld be easier if this is a struct instead of a class, and you can create instances using `{ }` without adding explicit constructors

Reply:
Done, it this what you had in mind?